src/shared/ifdef.rs

```diff
@@ -0,0 +1,23 @@
+// Copyright © 2015-2017 winapi-rs developers
```

Comment:
Shouldn't the copyright year be just 2018?

Reply:
Fixed